### PR TITLE
qa/tasks:  use qemu redir option and few other updates

### PR DESCRIPTION
**Pull request by vasukulkarni**
use qemu redir option and elminate the need to setup nfs mounts, also updates the image and points to latest xfs wrapper.

### CODE REVIEW

File: qa/tasks/userdata_setup.yaml [hunk 1]
```diff
@@ -1,25 +1,32 @@
-#cloud-config-archive
+#cloud-config
 
-- type: text/cloud-config
-  content: |
-    output:
-      all: '| tee -a /var/log/cloud-init-output.log'
+packages:
+  - xfslibs-dev
```
Comment (dillaman) [inline qa/tasks/userdata_setup.yaml]: Way to specific -- the QEMU task shouldn't be designed to handle only xfstest cases

**Comment by dillaman** [comment]
@vasukulkarni ping -- are you still planning to work on this?

**Comment by vasukulkarni** [comment]
@dillaman sorry, I will look into this shortly.

**Comment by dillaman** [comment]
please reopen once updates are available.

**Pull request closed without merging**